content/utilities/top10/index.md

```diff
@@ -0,0 +1,6 @@
+---
+title: Top10
```

Comment:
maybe worth clarifying,since youre not using the official api
```suggestion
title: Unofficial Top10
```

Reply:
very experimental, keep track of score at your own risk :)